### ISSUE TITLE
Add 20% held out test set to training code

### ISSUE THREAD

**Issue by zspald** [coganlab/RNN_phoneme_decoding_micro#31]
Use shuffle split with random seed as in hyperparameter optimization code
